api/src/pcapi/core/subscription/api.py

```diff
@@ -381,6 +381,16 @@ def get_allowed_identity_check_methods(user: users_models.User) -> list[models.I
     return allowed_methods
 
 
+def get_stepper_includes_phone_validation(user: users_models.User) -> bool:
```

Comment:
`is_phone_validation_in_app_stepper` 
-> ça retourne un booléen donc "is" ou "has" ou "does"  est plus approprié.
-> ça insiste sur le fait que c'est un affichage côté app